Organized rake tasks into separate files

clintonb opened the pull request:
Each namespace now resides in its own file, instead of cluttering Rakefile.

clintonb commented:
@macdiesel @wajeeha-khalid @BenjiLee please review

macdiesel commented:
@clintonb I checked out your brach to verify the namespaces have remained intact and I received this message:

```
➜  cs_comments_service git:(clintonb/organize-rake-tasks) bundle exec rake i18n:pull
rake aborted!
LoadError: cannot load such file -- factory_girl
/Users/brianbeggs/workspace/cs_comments_service/lib/tasks/db.rake:1:in `require'
/Users/brianbeggs/workspace/cs_comments_service/lib/tasks/db.rake:1:in `<top (required)>'
(See full trace by running task with --trace)
```

robrap commented:
Reviewers: I'm not sure if there is a simpler way to do this, but I added a separate temporary PR to see the changes made inside the newly refactored rake files, by putting them back into the main Rakefile:
https://github.com/edx/cs_comments_service/pull/169/files

robrap commented on lib/tasks/db.rake:
```diff
@@ -0,0 +1,204 @@
+
+    beginning_time = Time.now
+
+    create_list(:user, 10)
```
This looks like the test users will be slightly different than create_test_user.  Will they have the same username?  Does that matter?

robrap replied:
I now see where this is set up in the other unit testing PR.

clintonb commented:
I have simplified this PR to solely move the namespaces to separate files. The other changes are on another branch, and can be reviewed after we've added factories.

wajeeha-khalid commented:
So with the new change @robrap 's PR could not be compared against this PR; as this is only organising code. Is this true?

clintonb commented:
@wajeeha-khalid that is correct.

clintonb commented:
Any other feedback? Approvals?

robrap commented:
@clintonb In general seems fine, but I was waiting for a response to @macdiesel's failure.  Was that worked out offline?  Thanks.

clintonb commented:
@robrap @macdiesel's failure was due to my doing a little too much with this change. In addition to re-organizing the code, I also took steps to improve it by using factory_girl to generate seed data. This also resulted in your needing a separate PR to see these "improvements". Rather than deal with the complexity, I updated this PR to *only* reorganize the existing code.

Aside from a couple tasks that didn't work before I started working in this codebase, all of the tasks function as expected using the original code.

macdiesel commented:
:+1: thank you.

robrap commented:
:+1: I wasn't sure if the simplification resolved the issue.  Thanks.

clintonb commented:
Thanks, guys!
